devlib/native/win_native.cpp

```diff
@@ -433,14 +435,21 @@ std::vector<std::tuple<int, int, QString, QString>>
                 return;
             }
 
-            auto devInfo = winutil::getDevInfo(deviceProperties.containerId);
+            auto usbDevProperties = usbDevicesByContainerIdsMap.value(deviceProperties.containerId);
+            if (usbDevProperties.instanceId.isEmpty()) {
+                qCritical() << "Unable to find USB device using containerId";
+                return;
+            }
+
+            auto devId = winutil::extractDevPidVidInfo(usbDevProperties.instanceId);
+            auto usbPortPath = winutil::extractUsbPortPath(usbDevProperties.locationPath);
             auto deviceFilePath = winutil::nameFromDriveNumber(driveNum);
 
             devicesList.emplace_back(
-                devInfo.devId.vid,
-                devInfo.devId.pid,
+                devId.vid,
+                devId.pid,
                 deviceFilePath,
-                devInfo.usbPortPath
+                usbPortPath
```

Comment:
I hope that you're aware that all these values are going to be copied

Reply:
Changed to `std::move`

Reply:
I didn't actually insist on `std::move` just wanted to make things clear. And so that you know there's no point in moving primitive types like `int`, there's no performance improvement, but your code gets a bit clumsier because of all these `std::move`s

Reply:
Ok